src/main/java/com/landa44/simplerpg/attribute/AttributeService.java

```diff
@@ -0,0 +1,31 @@
+package com.landa44.simplerpg.attribute;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AttributeService {
+    private final AttributeRepository attributeRepository;
+
+    @Autowired
+    public AttributeService(AttributeRepository attributeRepository) {
+        this.attributeRepository = attributeRepository;
+    }
+
+    public List<Attribute> getAttributes() {
+        return attributeRepository.findAll();
+    }
+
+    public Attribute getLifeAttribute() {
+        return attributeRepository.getReferenceById(1L);
+    }
+
+    public Attribute getStrengthAttribute() {
+        return attributeRepository.getReferenceById(2L);
+    }
+
+    public Attribute getManaAttribute() {
+        return attributeRepository.getReferenceById(3L);
+    }
```

Comment:
Not a fan of this, you shouldn't rely on the ids

Reply:
moved to enum logic